### DISCUSSION TITLE
🙋 DSMR wil niet meer starten na docker container update

### DISCUSSION THREAD

**Discussion by bobvmierlo** [dsmrreader/dsmr-reader#1606] [Questions & answers / How-to's] [answered]
### Description

DSMR wil niet meer starten na docker container update, logging geeft weinig info helaas.
Ik draai DSMR al ruim 2 jaar als docker container op mijn Synology NAS. Vandaag toevallig na een hele lange tijd weer eens ingelogd en toen zag ik dat ik een paar versies had gemist. Dus ik update mijn Docker container, maar daarna wil hij niet meer starten. Ik heb de logging die ik eruit kan krijgen met mijn beperkte skills hier onder geplaatst.

Mijn P1 reader is trouwens niet verbonden via USB maar via het netwerk.

### DSMR-reader version

4

### DSMR-reader platform

Docker (e.g. Xirixiz)

### Debug info dump

```shell
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 10-adduser: executing... 
usermod: no changes
-------------------------
User UID: 803
User GID: 803
-------------------------
[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 20-set-app-defaults: executing... 
[ INFO ] DSMR Reader release: 5.1.0
[ INFO ] DSMR Reader Docker release: 2022.02.03
[ INFO ] Ensure permissions...
[ INFO ] Setting architecture requirements...
[ INFO ] X64 Architecture
[ INFO ] Fixing /dev/ttyUSB* security...
[ INFO ] Verifying if the DSMR web credential variables have been set...
[ INFO ] Verifying database connectivity to host: dsmrdb with port: 5432...
[ INFO ] Database connectivity successfully verified!
[ INFO ] Running post configuration...
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

**Comment by dennissiemensma** [comment, marked as answer]
Je kunt dit het beste voorleggen in het project van Xirixiz: https://github.com/xirixiz/dsmr-reader-docker/issues
Wellicht heeft hij nog suggesties om meer logging te tonen.
